client/src/components/snippet.jsx

```diff
@@ -1,24 +1,26 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+import ReactStars from 'react-stars';
 
 class Snippet extends Component {
   render() {
-    const url = "#/courses/" + this.props.data.id
+    const { id, name, rating, description } = this.props.data;
+    const url = "#/courses/" + id
     return (
       <a href={url}>
       <div className="snippet">
         <div className="snippet-name">
-          <h4>{this.props.data.name}</h4>
+          <h4>{name}</h4>
         </div>
         <div className="snippet-rating">
-          <p>Reviews: {this.props.data.rating}</p>
+          <ReactStars value={rating} size={20} edit={false}/>
```

Comment:
Here in snippet.jsx, keeping ReactStars this way (and not as a subcomponent) makes sense to me since the view will always be the same (no need to toggle based on loggedin  status / enrolled status).